src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -98,7 +98,6 @@
 				</When>
 			</Choose>
 			<ItemGroup>
-				<PackageReference Include="Xamarin.Google.Android.Material" Version="1.7.0.1" />
```

Comment:
I think this may have been required why are we removing it?

Reply:
there was a comment saying it was included in net 7 and that it wasnt needed